README.md

```diff
@@ -49,12 +49,6 @@ quark -a sample.apk -s -w quark_report.html
 
 ![](https://i.imgur.com/hG3Ag8t.png)
 
-## Available In
-
-<img src="https://i.imgur.com/nz4m8kr.png"/>
-
-[![asciicast](https://asciinema.org/a/416810.svg)](https://asciinema.org/a/416810)
-
```

Comment:
The change is similar to the one in #349. Since we have merged that PR, could you help remove the deletion here? Thanks!